var/spack/repos/builtin/packages/gdal/package.py

```diff
@@ -41,9 +41,10 @@ class Gdal(AutotoolsPackage):
     list_url   = "http://download.osgeo.org/gdal/"
     list_depth = 1
 
-    version('2.3.0', '2fe9d64fcd9dc37645940df020d3e200')
-    version('2.1.2', 'ae85b78888514c75e813d658cac9478e')
-    version('2.0.2', '940208e737c87d31a90eaae43d0efd65')
+    version('2.3.0',  '2fe9d64fcd9dc37645940df020d3e200')
+    version('2.1.2',  'ae85b78888514c75e813d658cac9478e')
+    version('2.0.2',  '940208e737c87d31a90eaae43d0efd65')
+    version('1.11.5', '5fcee5622430fbeb25556a4d07c06dd7')
```

Comment:
Does this old version of GDAL install correctly? I just overhauled the GDAL package last week so I want to make sure I didn't break too much. Hopefully most of the configure flags haven't changed much over the releases.

Reply:
Seems like I tested this before your recent overhaul and now it does not compile anymore. :-( I will investigate.

Reply:
I have updated the PR to make GDAL work.

There was only one mandatory change: gdal@1.11.5 does not support json-c@0.13.1 (but json-c@0.12.1 does not compile with recent compilers), so I simply removed the external dependency to make gdal fall back on its internal json-c copy.

I also disabled a few dependencies and moved around a few arguments for stuff that gdal@1.11.5 does not support.

Reply:
> gdal@1.11.5 does not support json-c@0.13.1 (but json-c@0.12.1 does not compile with recent compilers), so I simply removed the external dependency to make gdal fall back on its internal json-c copy.

Any idea what version its internal json-c copy is? I would prefer not to rely on internal copies when possible.

Reply:
Looks like 0.11. There is also a patch to support json-c 0.13.1 in gdal but it does not easily apply to gdal 1.x. I can give it another try, though.

Reply:
Turns out it was trivial to get json-c@0.12.1 to build with recent compilers (it's great to rediscover your own fixes in packages).